wazuh/wazuh-dashboard-plugins
# Split index dashboard management menu

### Description
This PR separates the index/dashboard management menu into indexer management and dashboard management.
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/6357

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/0513bf09-9206-4e9c-bec5-149de3c17356)

### Test

1. Open side bar menú.
2. Check de options in menus: `Dashboard management` and `Indexer management`.
3. Navigate every option and check if page is redirect.

### Check List
